### DISCUSSION TITLE
Pagination empty state

### DISCUSSION THREAD

**Discussion by rosemaryluo** [twilio-labs/paste#2280] [Q&A] [answered]
Hi, I have a view where normally there's multiple pages of content, but when filtered, we can end up with just a single page. If we use the standard paste component, do we have guidance for an empty state? Do we just have the pagination disappear?

**Comment by shleewhite** [comment, marked as answer]
Hi @rosemaryluo,

You can just use the "No results found" [empty state illustration](https://paste.twilio.design/patterns/empty-state/#examples-of-empty-state-use-cases) and hide the pagination component.

Please let me know if you have any other questions!

**Reply by rosemaryluo** [reply]
I meant we do have results, but only on a single page. I'll hide the pagination component for that!
